landoapi/auth.py

```diff
@@ -493,6 +494,43 @@ def __call__(self, f: Callable) -> Callable:
         return self._require_access_token(f)
 
 
+def ensure_user_has_scm_level(auth0_user: A0User, access_group: AccessGroup):
+    """Raise an appropriate `ProblemException` if the user is missing `scm_level_1`."""
+    # Return appropriate error message if user does not have commit access.
+    if not auth0_user.is_in_groups(access_group.membership_group):
+        raise ProblemException(
+            401,
```

Comment:
This should probably be  a `403`, if a user has provided a valid token, but the token does not give them the correct permissions.